docs/Displaying a User Interface/Alerts and Subtle Alerts/index.md

```diff
@@ -489,7 +489,7 @@ subtleAlert.softButtons = [button1]
 @![android,javaSE,javaEE]
 ```java
 // Soft buttons
-final int softButtonId = 123; // Set it to any unique ID
+final int softButtonId = 10001; // Set it to any unique ID
```

Comment:
Do we want to update the iOS examples or at least note in the iOS examples that if they set the id value to use 10001 or higher?